### DISCUSSION TITLE
Velero not able to restore pvc which are stored in efs

### DISCUSSION THREAD

**Discussion by mahi402** [vmware-tanzu/velero#6944] [Community support Q&A]
HI,

I have deployed my application in EKS WITH pvc in efs, when i take backup and try to restore the whole namespace,
pvc is getting restored fine, but some data and files are missing inside restored pvc, can you please help?

I deployed velero server using helm charts  velero/velero-plugin-for-aws:v1.7.0

**Comment by draghuram** [comment]
Just to confirm, you did file system backup?

**Reply by mahi402** [reply]
I took backup of namespace in EKS, using below command
velero backup create $SITENAME-ns-$NOW --storage-location=$SITENAME-ns --include-namespaces $SITENAME-ns  --ttl 1440h0m0s --wait
